views/idea.handlebars

```diff
@@ -1,8 +1,16 @@
-<br>
-<br>
-Name: {{idea.name}}
-<button>Like</button>
-<button>Interest</button>
+
+<div id="ideas" class="row g-3 align-items-center">
+    <div class="col">
+        <h3 class="fs-4 m-0">{{idea.name}}</h3>
+    </div>
+    <div class="col-auto">
+        <button class='btn btn-primary'>Like</button>
+    </div>
+    <div class="col-auto">
+        <button class='btn btn-secondary interest' value="{{idea.id}}">{{idea.interests.length}} Interested</button>
```

Comment:
We'll need a way here to determine if the current user viewing the page has marked this idea with interest, so we can add a data attribute the JS can use to choose between sending a POST vs DELETE request.

We'll likely need a helper to do this. I've got some example code in my flashcard project we can pull from:
The helper method  in `helpers.js` will looking something like this:
```
is_interested: (interestList, userId) => {
    return /** JS to get a true/false result on if userId is in the interestList **/;
}
```

And combing the custom helper with the built in `#IF` will look something like this.
```
{{#if (is_interested interestList loggedInUserId)}}content to conditionally add if current userId is in the list{{/if}}
```

Reply:
Thanks! That's a much cleaner way to code it--I'll work on that today.